File: src/main/java/com/gmail/hexragon/gn4rBot/command/admin/ArgsTestCommand.java
package com.gmail.hexragon.gn4rBot.command.admin;

import com.gmail.hexragon.gn4rBot.managers.commands.Command;
import com.gmail.hexragon.gn4rBot.managers.commands.CommandExecutor;
import com.gmail.hexragon.gn4rBot.managers.users.PermissionLevel;
import com.gmail.hexragon.gn4rBot.util.GnarMessage;
import org.apache.commons.lang3.StringUtils;

@Command(
		aliases = "args",
		permissionRequired = PermissionLevel.BOT_MASTER,
		showInHelp = false
)
public class ArgsTestCommand extends CommandExecutor
{
	@Override
	public void execute(GnarMessage message, String[] args)
	{
		message.reply("Args: `[" + StringUtils.join(args, ", ") + "]`.");
	}
}

File: src/main/java/com/gmail/hexragon/gn4rBot/command/admin/ArgsTestCommand.kt
package com.gmail.hexragon.gn4rBot.command.admin

import com.gmail.hexragon.gn4rBot.managers.commands.CommandExecutor
import com.gmail.hexragon.gn4rBot.managers.commands.annotations.Command
import com.gmail.hexragon.gn4rBot.managers.users.PermissionLevel
import com.gmail.hexragon.gn4rBot.util.GnarMessage

@Command(aliases = arrayOf("args"), permissionRequired = PermissionLevel.BOT_MASTER, showInHelp = false)
class ArgsTestCommand : CommandExecutor()
{
	override fun execute(message : GnarMessage?, args : Array<out String>?)
	{
		message?.reply("Args: `${args?.joinToString(", ")}`.")
	}
}